src/ProjectNeon/Assets/Features/Effects/Attack.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Events;
+
+public sealed class Attack 
+{
+    public Member Attacker { get; }
+    public Target Target { get; }
+
+    public Attack(Member attacker, Target target)
```

Comment:
This class would benefit from a secondary `ctor` that takes a Member as the second argument and converts them to a target, so that not all callers have to do that in the tests.

Reply:
@SilasReinagel I've refactored the class and changed the `Attack` class to store `Member` instead of `Target`